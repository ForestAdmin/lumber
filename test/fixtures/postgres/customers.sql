create table customers (
  id serial constraint customers_pk primary key,
  name varchar not null,
  description text,
  is_active boolean default true not null,
  created_at timestamp not null,
  updated_at timestamp
);
