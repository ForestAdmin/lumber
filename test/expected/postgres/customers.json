{
  "fields": [
    {
      "name": "name",
      "nameColumn": "name",
      "type": "STRING",
      "primaryKey": false,
      "defaultValue": null
    },
    {
      "name": "description",
      "nameColumn": "description",
      "type": "STRING",
      "primaryKey": false,
      "defaultValue": null
    },
    {
      "name": "isActive",
      "nameColumn": "is_active",
      "type": "BOOLEAN",
      "primaryKey": false,
      "defaultValue": true
    },
    {
      "name": "createdAt",
      "nameColumn": "created_at",
      "type": "DATE",
      "primaryKey": false,
      "defaultValue": null
    },
    {
      "name": "updatedAt",
      "nameColumn": "updated_at",
      "type": "DATE",
      "primaryKey": false,
      "defaultValue": null
    }
  ],
  "references": [],
  "primaryKeys": [
    "id"
  ],
  "options": {
    "underscored": true,
    "timestamps": true,
    "hasIdColumn": true,
    "hasPrimaryKeys": true
  }
}
